borsh/src/schema.rs

```diff
@@ -712,4 +856,61 @@ mod tests {
         let phantom_declaration = PhantomData::<Vec<u8>>::declaration();
         assert_eq!("nil", phantom_declaration);
     }
+
+    mod test_structs {
+        #[cfg(not(feature = "std"))]
```

Comment:
there's no need to use the `#[cfg(...)]` clause, if the dispatch is done with `crate::__private::maybestd` import

Reply:
I had troubles getting this to work somehow and an top of it couldn’t reproduce the failures locally.  I got the code to state where it passes CI and passes on my computer locally so I think I’m sticking with it. ;)